eng/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: Repository
+    type: string
+    default: $(Build.Repository.Name)
+  - name: Prefix
+    type: string
+  - name: CIConventionOptions
+    type: string
+    default: ''
+  - name: UPConventionOptions
```

Comment:
For my own context, what is the UP acronym?

Reply:
unified pipeline